packages/web-api/src/WebClient.spec.ts

```diff
@@ -463,14 +463,13 @@ describe('WebClient', () => {
         const scope = nock('https://slack.com', {
           reqheaders: {
             'User-Agent': (value) => {
+              // User Agent value is different across platforms.
+              // on mac this is: @slack:web-api/7.7.0 node/18.15.0 darwin/23.6.0
+              // on windows this is: @slack:web-api/7.7.0 cmd.exe /22.10.0 win32/10.0.20348
```

Comment:
Thanks for the detailed documentation. These kinda of notes help future maintainers a lot!